### ISSUE TITLE
Is it is possible and useful to print 3D ventilator parts?

### ISSUE THREAD

**Issue by ajlennon** [DoESLiverpool/covid19#22]
Kicking off this issue with this tweet:

ref: https://twitter.com/jeromelebel/status/1244052594202480645?s=20

![image](https://user-images.githubusercontent.com/1537834/77836960-24180680-7153-11ea-9c6b-c069ea63449a.png)

ref: https://www.premium-forum.fr/viewtopic.php?f=5&t=2199

**Comment by ajlennon** [comment]
From Ed Saul 

>Oxford and KCL design a new open-source ventilator
>The OxVent team, a group of engineers and medics from Oxford University and Kings College London, has been shortlisted by government to go to the next stage ...

[https://www.youtube.com/watch?v=tpOST90lbj4](https://www.youtube.com/watch?v=tpOST90lbj4)

**Comment by plastictactics** [comment]
I assume there is even more risk regarding CE Marking and intellectual property with ventilator components, and that we already have our hands full with visors. As you might have noticed I'm closing as many issues as I reasonably can, to make it easier for people to engage with the active issues - OK to close this one for now?

**Comment by drakard** [comment]
I have had little involvement there for the last couple of weeks but I have seen Warrington Fab Lab have apparently (based on their social media) been having good success printing valves for Decathlon masks (which are being used by local health providers now). Since we are laser cutting visors, and 3d printers are much slower than laser cut for making visors, and these valves might be 'higher value' (?) than visors, could there be an advantage to printing them instead? Apparently they have donated at least 75 of those valves to make ventilator masks so far.

**Comment by ajlennon** [comment]
@drakard great point - I have a number of people asking about this. If you want to champion this I'll introduce

**Comment by drakard** [comment]
@ajlennon I will contact Fab Lab and see what information I can get

**Comment by drakard** [comment]
@ajlennon I have spoken with the Fab Lab. Do you have someone on the medical side asking about this? Ideally someone like a consultant etc. who can make the use decision? Or just people who want to help supply?

**Comment by drakard** [comment]
@afroleft I'm sorry, I do not know if this is appropriate to address to you, I am just guessing at people's roles. Are you dealing with any medical staff / managers who would might want to have valve adaptors to fit Decathlon masks for ventilators? The Fab Lab in Warrington have apparently had considerable success with this.

**Comment by afroleft** [comment]
> Are you dealing with any medical staff / managers who would might want to have valve adaptors to fit Decathlon masks for ventilators?

Yes have access to medical folks and procurement teams but I have no line of sight on the ventilator situation. 

What exactly is it you were hoping to do?

**Comment by drakard** [comment]
@afroleft I am just trying to find if there is any desire for extra ventilator masks of the kind that can be made from a Decathlon mask and a 3d printed valve. If there is it seems like these valves might be a 'higher value' product for 3d printing a small amount of. Since the Fab Lab in Warrington have already validated them https://d2g8igdw686xgo.cloudfront.net/47199534_158610850338017_r.jpeg  (they have apparently shipped around 75 so far) if there is demand I would guess it would be much more straightforward than starting from scratch.

**Comment by afroleft** [comment]
@drakard ok, assuming the next step is to reach out to NHS Trust procurement teams, you want to draft up a note that I can drop in to an email to a few of the folks that we're dealing with? 

I'll send it on. Fingers crossed sourcing teams can either answer your question or at least signpost us to someone helpful.

**Comment by drakard** [comment]
@afroleft Thank you very much. It could be something like this:

> We have been looking for more ways we can help, and one possibility which appears to have had some success is the adaption of full face snorkelling masks for medical use. There are two uses these could be adapted for. Firstly, they can be adapted to provide masks for ventilators. This 1 minute video shows an example of that use: https://www.youtube.com/watch?v=n6qQLsUa7jk . Secondly, they can be adapted to provide a full face mask for medical practitioners. This 1 minute video shows an example of that use: https://www.youtube.com/watch?v=q6EByykKUU8 . An organisation in Warrington has been reporting success in providing masks and adaptors for the first use to intensive care units in Manchester. if they are desired we could potentially source donations of masks together with manufacturing the adaptors. At this stage we would just like to know if you have a potential interest in these options.

**Comment by drakard** [comment]
@ajlennon raised the concern to me that this kind of full face mask is associated with dangers (in normal use). Serious risks associated with these masks have been questioned in the media:
- https://www.cbsnews.com/news/hawaii-full-face-snorkel-mask-related-deaths/
- https://hawaiioceanproject.com/why-we-no-longer-support-full-faced-snorkel-masks/
- https://www.youtube.com/watch?v=Biw5CijcbFQ

These considerations must assessed if this kind of mask is to be used.

**Comment by afroleft** [comment]
@drakard reaching out to a few friends that are ICU medics down at Guys in London. Will feedback.

**Comment by afroleft** [comment]
They don’t use them at Guys & St Thomas’s. Also they are wondering if this piece is the constraint, or, is the constraint the machine itself. 

Will try a few NHS central sourcing teams to see what they think

**Comment by drakard** [comment]
@afroleft thank you.
